code/processes/dqe.q

```diff
@@ -1,10 +1,39 @@
 \d .dqe
 
+configcsv:@[value;`.dqe.configcsv;first .proc.getconfigfile["dqeconfig.csv"]];
+
 init:{
   .lg.o[`init;"searching for servers"];
   .servers.startup[];                                                                                           /- Open connection to discovery
   }
 
+configtable:(
+
+
+  action:`symbol$();              // type of check
+
```

Comment:
there's a lot of empty lines in this schema, is that necessary?

more debatable, I think most of these column names are reasonably self-explanatory, could this just be a one-liner schema perhaps?

Reply:
thanks, going to delete empty lines and make it one line